metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -463,9 +464,22 @@ def _extract_record(self, topic: str) -> Iterable[MetadataWorkUnit]:  # noqa: C9
         self.report.report_workunit(wu)
         yield wu
 
+        # 5. Add the subtype aspect marking this as a "topic"
+        subtype_wu = MetadataWorkUnit(
+            id=f"{topic}-subtype",
+            mcp=MetadataChangeProposalWrapper(
+                entityType="dataset",
+                changeType=ChangeTypeClass.UPSERT,
+                entityUrn=dataset_urn,
+                aspectName="subTypes",
+                aspect=SubTypesClass(typeNames=["topic"]),
```

Comment:
In the long term, I think these should be constants as it is safer to use in multiple places and easier to refactor if needed

Reply:
agreed.. will do this in a follow-up PR.